CONTRIBUTING.md

```diff
@@ -25,6 +28,41 @@ We also maintain a [gitter chat room][link_gitter] for more informal conversatio
 There is significant cross-talk between these two spaces, and we look forward to hearing from you in either venue!
 As a reminder, we expect all contributions to `tedana` to adhere to our [code of conduct][link_coc].
 
+<img align="right" width="50%" src="https://linuxcentro.com.br/wp-content/uploads/2017/04/github-520x350.jpg" alt="Two github cats working together"/>
+
+## Contributing through GitHub
+
+[git][link_git] is a really useful tool for version control. [GitHub][link_github] sits on top of git and supports collaborative and distributed working.
+
+We know that it can be daunting to start using git and GitHub if you haven't worked with them in the past, but the BIDS Starter Kit maintainers are here to help you figure out any of the jargon or confusing instructions you encounter! :heart:
```

Comment:
I'm going to update this and restructure a bit of the text before merging, per our discussion on Slack ! thanks so much for adding this :sparkles: :hugs: